pkg/sql/sem/tree/copy.go

```diff
@@ -90,17 +95,20 @@ func (o *CopyOptions) Format(ctx *FmtCtx) {
 	var addSep bool
 	maybeAddSep := func() {
 		if addSep {
-			ctx.WriteString(" ")
+			ctx.WriteString(", ")
 		}
 		addSep = true
 	}
+	ctx.WriteString("(")
```

Comment:
do you think we should add the `WITH ` for completeness, or nah?

Reply:
yeah, seems more clear

Reply:
ah wait, it's already there https://github.com/cockroachdb/cockroach/blob/20e2adda3c76c7172dd986c871df0ae9a346918f/pkg/sql/sem/tree/copy.go#L51

Reply:
d'oh